### DISCUSSION TITLE
Size of data model

### DISCUSSION THREAD

**Discussion by bendfeldt** [DaxStudio/DaxStudio#728] [Q&A] [answered]
Hi all,

I've looked into the DMV and also the VertiPaq Analyzer. 
In that regard i've seen from SQLBI that the "Col Size" in VertiPaq Analyzer is the sum of "Data", "Dictionary" and "Heir Size".
I can find the columns "Data" and "Dictionary" but not the "Heir Size".

Does anyone know where to find the "Heir Size" in the DMV?
I've found "Data" in the DMV  -> select * from $SYSTEM.DISCOVER_STORAGE_TABLE_COLUMN_SEGMENTS
I've found "Dictionary" in the DMV -> select * from $SYSTEM.TMSCHEMA_COLUMN_STORAGES

Hope you can help
![Screenshot 2021-09-29 090355](https://user-images.githubusercontent.com/91461181/135219319-afeba622-1c36-4f11-96fe-9398234cf19f.jpg)

**Comment by marcosqlbi** [comment, marked as answer]
Look at the source code (which include DMVs) for Hierarchies Size.
Column hierarchies: https://github.com/sql-bi/VertiPaq-Analyzer/blob/master/Dax.Model.Extractor/DmvExtractor.cs#L561
User hierarchies: https://github.com/sql-bi/VertiPaq-Analyzer/blob/master/Dax.Model.Extractor/DmvExtractor.cs#L619

**Reply by bendfeldt** [reply]
Thanks a lot.. this really helhed me out a lot
